packages/sdk/src/controllers/SubscriberController.ts

```diff
@@ -66,22 +66,47 @@ export class SubscriberController {
         callBack && callBack(unit as MeasurementUnit);
     }
 
+
     /**
-     * Listener on the state of the currently selected frame, if this changes, this listener will get triggered with the updates
-     * @param frameLayout Stringified object of FrameLayoutType
+     * @deprecated use `onSelectedFramesLayoutChanged` instead
+     * Listener on the state of the currently selected frames
+     * Whenever multiple frames are selected, it always will be the first selected `FrameLayoutType`
+     * @param frameLayout Stringified array of `FrameLayoutType` objects
      */
     onSelectedFrameLayoutChanged = (frameLayout: string) => {
-        const callBack = this.config.onSelectedFrameLayoutChanged;
-        callBack && callBack(JSON.parse(frameLayout));
+        const callBack = this.config.onSelectedFramesLayoutChanged;
+        const frames = JSON.parse(frameLayout);
+        callBack && callBack(frames[0]);
+    };
```

Comment:
why deprecate it? it is still a valid call

Reply:
The engine-side update now exposes an array of frames instead of just one. I think it's best to avoid embedding specific display logic within the SDK. Instead, giving integrators the choice to decide how they want to present this data — whether focusing on a single frame's properties or not.

Reply:
My question is not persé why you deprecate it, I know why, but I don't feel like this is necessary.
The behaviour of the call, still stays the same so the function is still a valid one imo. Since we use the new call with index 0, this is perfect behaviour and one breaking change less to worry about?

Reply:
the behavior of the call remains the same since there is a workaround for that (to avoid breaking changes in the first place). I don't think that is a correct approach and still convinced that it shouldn't be there. No strong opinion though

Reply:
yes you might be right, I'll resolve convos again